proto/recoverdatapb.proto

```diff
@@ -0,0 +1,56 @@
+syntax = "proto3";
+package recover_data;
+
+import "gogoproto/gogo.proto";
+import "rustproto.proto";
+
+option (gogoproto.sizer_all) = true;
+option (gogoproto.marshaler_all) = true;
+option (gogoproto.unmarshaler_all) = true;
+option (rustproto.lite_runtime_all) = true;
+
+// request to read region meata from a store
+message ReadRegionMetaRequest {
+    uint64 store_id = 1;
+}
+
+message RegionMeta {
+    uint64 region_id = 1;
+    uint64 last_log_term = 2;
+    uint64 last_index = 3;
+    uint64 commit_index = 4;
+    uint64 version = 5;
+    bool tombstone = 6; //reserved, it may be used in late phase for peer check
```

Comment:
As we discussed, PD needs information to get a max allocated ID. So
1. `peer_id` should be added in to `RegionMeta`;
2. When recovering, TiKV must report all regions' information to PD, includes tombstone replica.

Then PD can calculate MaxAllocID = max(max(region_ids), max(peer_ids)).

Reply:
it could be a good idea, will add peer id